decidim-proposals/spec/features/proposals_spec.rb

```diff
@@ -408,6 +408,20 @@
       expect(page).to have_css(".card--proposal", count: 3)
     end
 
+    it "lists the proposals ordered randomly by default" do
+      allow_any_instance_of(Decidim::Proposals::Proposal::ActiveRecord_Relation).to \
+        receive(:order_randomly) { |scope, _seed| scope.order(title: :asc) }
+
+      lucky_proposal = create(:proposal, title: "A", feature: feature)
+      unlucky_proposal = create(:proposal, title: "B", feature: feature)
+
+      visit_feature
+
+      expect(page).to have_selector("a", text: "Random")
+      expect(page).to have_selector("#proposals .card-grid .column:first-child", text: lucky_proposal.title)
+      expect(page).to have_selector("#proposals .card-grid .column:last-child", text: unlucky_proposal.title)
+    end
```

Comment:
I changed this test in a different way to get it passing in Rails 5.1. We might want to normalize both tests, maybe through a shared example.